pkg/enum/linear_test.go

```
@@ -0,0 +1,520 @@
+
+// Ensure types package is referenced so imports stay tidy.
+var _ types.Provenance = types.ExtendedProvenance{}
+
+func TestLinearGraphQL_Success(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "POST", r.Method)
+		assert.Equal(t, "application/json", r.Header.Get("Content-Type"))
+		assert.Equal(t, "lin_api_test", r.Header.Get("Authorization"))
+
+		var body map[string]interface{}
+		json.NewDecoder(r.Body).Decode(&body)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Decoder).Decode` is not checked

(errcheck)